app/assets/stylesheets/components.scss

```diff
@@ -3023,12 +3023,12 @@ button.icon-button.active i.fa-retweet {
 .media-gallery__item-thumbnail {
   background-position: center;
   background-repeat: no-repeat;
-  background-size: cover;
   cursor: zoom-in;
   display: block;
   height: 100%;
   text-decoration: none;
   width: 100%;
+  background-color: #191b22;
```

Comment:
I am not against adding a default background color to the thumbnails! In cases where they fail to load, such a thing would look a lot better. But you should use a color variable instead of a hardcoded value. See variables.scss for available ones. You can adjust the lightness with lighten()/darken() (see examples in the code), I think this is something like `darken($ui-base-color, 8%)`